plugins/org.eclipse.thym.core/src/org/eclipse/thym/core/HybridProject.java

```diff
@@ -147,6 +147,14 @@ public IFile getConfigFile(){
 		return null;
 	}
 	
+	public IFile getPlatformsJSONFile() {
```

Comment:
Currently, `getPlatformsJSONFile()` is used in `HybridMobileEngineManager`, since getting active engines from `platforms.json` is more accurate than reading `config.xml`. It needs to be public so that we can read the `platforms.json` file when necessary.

Alternatively, the method could be moved to `HybridMobileEngineManager`, but it feels like the functionality fits better with the purpose of HybridProject.